spec/features/admin/merchants/show_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Merchant Show Page' do
+  let!(:merchant_1) {Merchant.create!(name: 'Ron Swanson')}
+  let!(:merchant_2) {Merchant.create!(name: 'Leslie Knope')}
+
```

Comment:
`visit admin_merchant_path` runs at the beginning of every test. We could add after the let blocks and before the first scenario:
```
before(:each) do
   visit admin_merchant_path
end
```
to dry up the code a little bit.

Reply:
Nice, just implemented these before each routes on my local and will be updated when I push my next round of changes to main.

Reply:
Excellent!